src/components/Folder/index.jsx

```diff
@@ -1,24 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 
-import { Typography } from 'antd';
-import { FolderOutlined } from '@ant-design/icons';
+import { Card, Divider, Modal, Row, Typography } from 'antd';
+import { DeleteOutlined, FolderOutlined } from '@ant-design/icons';
 
 import './styles.scss';
 import routes from 'constants/routes';
 
 const { Text } = Typography;
 
 function Folder({ folderObj }) {
+  const [isModalVisible, setIsModalVisible] = useState(false);
   const history = useHistory();
 
   async function redirectPage() {
     history.push(routes.FOLDER, folderObj);
   }
+
+  function showModal() {
+    setIsModalVisible(true);
+  }
+
```

Comment:
For `handleOk` and `handleCancel` we can do them in 1 function. Let's name it `hideModal`